packages/components/pie-notification/src/index.ts

```diff
@@ -43,31 +44,31 @@ export * from './defs';
  */
 export class PieNotification extends LitElement implements NotificationProps {
     @property({ type: Boolean })
-    public isOpen = true;
+    public isOpen = defaultProps.isOpen;
 
     @property()
-    @validPropertyValues(componentSelector, variants, 'neutral')
-    public variant: NonNullable<NotificationProps['variant']> = 'neutral';
+    @validPropertyValues(componentSelector, variants, defaultProps.variant)
+    public variant: NonNullable<NotificationProps['variant']> = defaultProps.variant;
 
     @property()
-    @validPropertyValues(componentSelector, positions, 'inline-content')
-    public position: NonNullable<NotificationProps['position']> = 'inline-content';
+    @validPropertyValues(componentSelector, positions, defaultProps.position)
+    public position: NonNullable<NotificationProps['position']> = defaultProps.position;
 
     @property({ type: Boolean })
-    public isDismissible = true;
+    public isDismissible = defaultProps.isDismissible;
 
     @property({ type: Boolean })
-    public isCompact = false;
+    public isCompact = defaultProps.isCompact;
 
     @property({ type: String })
     public heading!: string;
 
     @property()
```

Comment:
I know this isn't related to your change, but should this property decorator have an arg of `{ type: String }` inside?

Reply:
Should I add an empty `heading` default prop and have that `!` removed?